wcivf/apps/elections/templates/elections/includes/_registration_details.html

```diff
@@ -0,0 +1,59 @@
+{% load postcode_tags %}
+{% load humanize %}
+
+<section class="card">
+
+  <h2 id='where'>
+    <span aria-hidden="true">🗳️</span>
+    Register to vote
+    <span aria-hidden="true">🗳️</span>
+  </h2>
+
+<p>You need to be registered in order to vote.
+  If you aren't registered to vote visit 
+  <a href="https://www.gov.uk/register-to-vote">
+    https://www.gov.uk/register-to-vote
+  </a>
+</p>
+<p>Register before midnight on {{ postelection.registration_deadline }} 
```

Comment:
This is duplicated - I guess because of the below question? But just thought I'd point out in case its been missed